ERCS/erc-7579.md

```diff
@@ -48,149 +48,158 @@ The smart account's `validateUserOp` function SHOULD return the return value of
 
 #### Execution Behavior
 
-To comply with this standard, smart accounts MUST implement the two interfaces below. If an account implementation elects to not support any of the execution methods, it MUST implement the function, but revert instead of complying with the specifications below. This is in order to avoid unpredictable behavior with fallbacks.
+To comply with this standard, smart accounts MUST implement the execution interface below:
 
 ```solidity
 interface IExecution {
     /**
-    * @dev packing batched transactions in a struct saves gas on both ERC-4337 and Executor Module flows
-    */
-    struct Execution {
-        address target;
-        uint256 value;
-        bytes callData;
-    }
+     * @dev Executes a transaction on behalf of the account.
```

Comment:
Open question: should we add natspec that is not specs in here or only in the interface under assets?

Reply:
[ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) includes natspec